crates/iota-indexer/src/apis/governance_api.rs

```diff
@@ -358,6 +452,42 @@ impl<T: R2D2Connection + 'static> GovernanceReadApi<T> {
             })
             .collect::<Vec<(IotaAddress, ObjectID, ObjectID, u64, bool)>>();
 
+        // Get inactive validator rate tables
+        for df in self
+            .inner
+            .get_dynamic_fields_in_blocking_task(
+                system_state_summary.validator_candidates_id,
+                None,
+                system_state_summary.validator_candidates_size as usize,
+            )
+            .await?
+        {
+            let raw_address = String::deserialize(df.name.value)
+                .map_err(|err| IotaError::from(err.to_string()))?;
+            let validator_address = IotaAddress::from_str(&raw_address)?;
+
+            let validator_candidates_id = system_state_summary.validator_candidates_id;
+
+            let validator_candidate = self
+                .inner
+                .spawn_blocking(move |this| {
+                    iota_types::iota_system_state::get_validator_from_table(
+                        &this,
+                        validator_candidates_id,
+                        &validator_address,
+                    )
+                })
+                .await?;
+
+            tables.push((
+                validator_candidate.iota_address,
+                validator_candidate.staking_pool_id,
+                validator_candidate.exchange_rates_id,
+                validator_candidate.exchange_rates_size,
+                false,
+            ));
+        }
```

Comment:
This seems like a block of code that can be reused for the `inactive` validators, so you might consider making it a method accepting the table id.

Reply:
Now fetching [Inactive](https://github.com/iotaledger/iota/pull/4313/commits/353da3b8f1c64730650f552d6b69a8a80833f0d9#diff-324f865071b593cb4ed59dd865b9600fae68065a818e622b14f4f5554812fd98R418-R436) and [Candidate](https://github.com/iotaledger/iota/pull/4313/commits/353da3b8f1c64730650f552d6b69a8a80833f0d9#diff-324f865071b593cb4ed59dd865b9600fae68065a818e622b14f4f5554812fd98R469-R491) validatros uses the utility function which unifies the logic